compiler-core/src/docs.rs

```diff
@@ -206,6 +214,19 @@ pub fn generate_html(
             url: format!("{}.html", module.name),
         });
 
+        let rendered_documentation = render_markdown(&module.ast.documentation.iter().join("\n"));
+        let rendered_text_documentation =
+            manipulate::trim(rendered_documentation._strip_tags().as_str(), "\n ");
+        let page_meta_description = &manipulate::replace_all(
+            manipulate::trim(
+                format!("{} · {}", &module.name, &rendered_text_documentation).as_str(),
+                "· \n",
+            )
+            .as_str(),
+            "\n",
+            " ",
+        );
```

Comment:
I like the idea of having a description, but the module doc comment could be thousands of lines long- I don't think it is suitable for this tag. What do other languages use here?

Reply:
We don't really have anything else. We can truncate after 320 (or 500) characters with one elipsis character?

> … aka U+2026

Reply:
> view-source:https://hexdocs.pm/phoenix/Phoenix.Naming.html

lacks stuff. the best we have is the module doc introduction.

Reply:
... I have pushed an update, the code may be not idiomatic and I couldn't make `format!()` work for now... but I'd rather have your opinion first if the direction is good for you.

Reply:
We need to look at other options here, the doc comment is not suitable for use as a description.